tests/suites/test_suite_gcm.function

```diff
@@ -8,6 +8,49 @@
  */
 
 /* BEGIN_CASE */
+void gcm_bad_parameters(  int cipher_id, int direction,
+                          char *hex_key_string, char *hex_src_string,
+                          char *hex_iv_string, char *hex_add_string,
+                          int tag_len_bits, int  gcm_result)
```

Comment:
Could you please add space before `)` and remove space between `int` and `gcm_result`?